app/views/account.py

```diff
@@ -90,9 +90,17 @@ def edit():
         form.reseller_name = account.reseller.name
         return render_template("account_details.html", form=form)
     else:
+        prev_product = None
+        prev_reseller = None
+        if 'prev_reseller' in request.args and 'prev_product' in request.args:
+            prev_product = request.args['prev_product']
+            prev_reseller = request.args['prev_reseller']
         form = AccountForm()
-        form.products = Product.query.all()
-        form.resellers = organize_list_starting_with_value(Reseller.query.order_by(Reseller.name).all(), 'NITRIX')
+        form.products = organize_list_starting_with_value(
+            Product.query.order_by(Product.name).all(), prev_product) if prev_product else Product.query.all()
+        form.resellers = organize_list_starting_with_value(
+            Reseller.query.order_by(Reseller.name).all(),
```

Comment:
Reseller also has field **deleted** - if it's value True - means this Reseller deleted.
So we don't show it in the list